dataregistry-extension/src/mimetypes.ts

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
I don't think we need a mimetypes file

Reply:
The reason I did was because I noticed MIME type magic constants which were repeated in various places (e.g., `if type === "text/csv" then...`).
​
Whenever I see repeated magic constants, I sense we'd be better off defining these in a single place (file) which we can then import.
​
Otherwise, we run the risk of typos and subtle bugs. For example, if I forget the `i` in `"text/plain"` when doing a comparison or when providing as an argument, TypeScript won't be able to catch this as a type error. I'd still be comparing strings or providing a string argument.
​
In this case, I'd only be able to catch the typo during runtime tests, not at compile time.
​
However, if I import a MIME types database and misspell a property name (most likely a name which does not exist), then I am more likely to get a compile error because I'd provide, e.g., `undefined` as an argument and not a string, and avoid the runtime bug.